docs/src/guides/pip.md

```diff
@@ -0,0 +1,336 @@
+---
+title: Build a python project with pip
+---
+
+!!! info
+
+    We recommend reading our [Getting Started](./getting-started.md) guide first if you have not done so yet!
+
+this guide we are going to take a look at two annotated examples using the [pip module](../reference/pip/index.md):
```

Comment:
Hate to point this out right after you pushed, but I think this missing an "In" at the beginning of the sentence!

Reply:
Haha, right. No worries, I'll fix it with the next PR, probably later "today" (I am in CEST).